CMakeLists.txt

```diff
@@ -7,9 +7,9 @@ endif()
 project(SignalEstimator)
 
 option(BUILD_GUI "build Qt GUI" ON)
-
 option(ENABLE_SANITIZERS "enable sanitizers" OFF)
 option(ENABLE_WERROR "enable -Werror" OFF)
+option(WITH_TESTS "build tests" ON)
```

Comment:
There is standard option for this - `BUILD_TESTING`.

Reply:
Thanks, didn't knew this!